src/home-page/index.js

```diff
@@ -208,7 +225,8 @@ const HomePage = () => {
                     <img className="image-height-300 float-start"
                          src={require("../images/planting.jpg")}/>
                 </div>
-                <a href="http://localhost:3000/opportunities" className="col-8 fs-2 fw-bold text-white d-flex align-items-center justify-content-center">
+                <a href="http://localhost:3000/opportunities"
+                   className="col-8 fs-2 fw-bold text-white d-flex align-items-center justify-content-center">
```

Comment:
Note that this is the same list of classes as the above link. To make it easy and readable to have lots of these links, we could make a styled component and re-use it, ex (in a separate file called StyledLink.tsx):

```tsx
export default const StyledLink = ({ path, title }) => {
  return (
    <a href={path} className='col-8 fs-2 fw-bold text-white d-flex align-items-center justify-content-center'>
      <h1>{title}</h1>
    </a>
  )
}
```

and then we'd just have in here
```tsx
<div>
  <StyledLink path={Routes.MAP} title='SUSTAINABILITY MAP' />
  <StyledLink path={Routes.OPPORTUNITIES} title='OPPORTUNITIES FOR STUDENTS' />
</div>
```